closed/OpenJ9.gmk

```diff
@@ -86,6 +86,22 @@ endif
 	stage_openj9_build_jdk \
 	#
 
+# jitserver
+
+ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+$(eval $(call SetupCopyFiles,OPENJ9_STAGED_FILES, \
+        DEST := $(JDK_OUTPUTDIR)/bin, \
+        SRC := $(OPENJ9_VM_BUILD_DIR), \
+        FILES := jitserver \
+        ))
+
+$(eval $(call SetupCopyFiles,OPENJ9_STAGED_FILES, \
+        DEST := $(JDK_OUTPUTDIR)/jre/bin, \
+        SRC := $(OPENJ9_VM_BUILD_DIR), \
+        FILES := jitserver \
+        ))
+endif
```

Comment:
Please use a tab to indent lines here, for consistency.
Also add a comment: `endif # OPENJ9_ENABLE_JITSERVER`.

Reply:
Updated the code as per the suggestions.